common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
   "message.voicechat.failed_to_send_ping": "Failed to send ping: %s",
   "message.voicechat.sending_ping": "Sending ping...",
   "message.voicechat.ping_sent_waiting": "Ping sent. Waiting for response...",
-  "message.voicechat.ping_received": "Got a response in %sms",
-  "message.voicechat.ping_received_attempt": "Got a response after %s attempts in %sms",
+  "message.voicechat.ping_received": "Got a response after %s ms",
+  "message.voicechat.ping_received_attempt": "Got a response after %s attempts and %s ms",
   "message.voicechat.ping_retry": "No response. Retrying...",
   "message.voicechat.ping_timed_out": "Request timed out after %s attempts",
   "message.voicechat.icons_hidden": "Voice chat icons hidden",
   "message.voicechat.icons_visible": "Voice chat icons visible",
-  "message.voicechat.incompatible_version": "Your voice chat version is not compatible with the servers version.\nPlease install version %s of %s.",
-  "message.voicechat.incompatible": "Your voice chat version is not compatible with the servers version.",
-  "message.voicechat.group_members": "Group members",
-  "message.voicechat.no_group_members": "This group is empty",
-  "message.voicechat.join_create_group": "Join or Create Group",
-  "message.voicechat.group_name": "Group name",
-  "message.voicechat.leave_group": "Leave group",
+  "message.voicechat.incompatible_version": "Your voice chat client version is not compatible with the server-side version.\nPlease install version %s of %s.",
+  "message.voicechat.incompatible": "Your voice chat client version is not compatible with the server-side version.",
+  "message.voicechat.group_members": "Group members:",
+  "message.voicechat.no_group_members": "Empty group",
+  "message.voicechat.join_create_group": "Join or Create a Group",
+  "message.voicechat.group_name": "Group Name",
+  "message.voicechat.leave_group": "Press to leave the group.",
   "message.voicechat.not_in_group": "You are not in a group",
   "message.voicechat.invite": "%s invited you to the group %s. %s",
   "message.voicechat.accept_invite": "ACCEPT",
-  "message.voicechat.accept_invite.hover": "Click to accept invitation",
-  "message.voicechat.join_successful": "Successfully joined %s",
+  "message.voicechat.accept_invite.hover": "Click to accept the invitation",
+  "message.voicechat.join_successful": "Successfully joined the group %s",
   "message.voicechat.leave_successful": "Successfully left the group",
   "message.voicechat.groups_disabled": "Groups are disabled on this server",
   "message.voicechat.recording_disabled": "Recording is disabled on this server",
   "message.voicechat.invite_successful": "Successfully invited %s",
   "message.voicechat.processing_recording_session": "Processing recording session...",
   "message.voicechat.processing_progress": "Processing %s%%",
-  "message.voicechat.save_session": "Saved session to '%s'",
+  "message.voicechat.save_session": "Saved session to %s",
```

Comment:
That's because the `%s` format specifier is replaced at runtime with the file path to the recording on disk. In such a case, vanilla Minecraft simply uses a different text style without any additional quotation marks.

In my opinion, the gray color of the path (something that you already do) is sufficient. You could optionally add an underline to further emphasize that the path is clickable, but that's it.

Reply:
Yeah thats fair. 

Isn't it already underlined? If not we need to change that

Reply:
I can do it. That way, you won't have to keep it at the back of your mind and then backport it all the way down to 1.12.2.

Reply:
Thanks!